Packages/com.pg649.creaturegenerator/Runtime/CreatureGenerator.cs

```diff
@@ -7,19 +7,21 @@
 
 public class CreatureGenerator
 {
-    public static GameObject ParametricBiped(CreatureGeneratorSettings settings, ParametricCreatureSettings creatureSettings,
-        int? seed)
+    public static GameObject ParametricBiped(CreatureGeneratorSettings settings,
+        ParametricCreatureSettings creatureSettings, int? seed,
+        Dictionary<(BoneCategory, BoneCategory), JointLimits> limitOverrides)
```

Comment:
I would use the JointLimitOverrides themselves as the parameter. Keeps the API more uniform and stops the user from having to convert the settings object into a dictionary.

I would also make the parameter optional. That way this isn't a breaking change and it is clearer than providing an empty override list.